File: comms/push-notifications/credentials/delete-credential/delete-credential.cs
using TwilioComms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioCommsClient(
            accountId: "<username>",
            authToken: "<password>"
        );

        await client.PushNotifications.Credentials.DeleteAsync(
            "comms_credential_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}

File: comms/push-notifications/credentials/delete-credential/delete-credential.php
<?php

namespace Example;

use Twilio\TwilioComms;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->pushNotifications->credentials->delete(
    'comms_credential_01h9krwprkeee8fzqspvwy6nq8',
);
